spec/features/admin/admin_claims_filtering_spec.rb

```diff
@@ -49,10 +49,12 @@
 
     click_on "View claims"
 
-    expect(page.find("table")).to have_content("ECP").exactly(10).times
-    expect(page.find("table")).to have_content("TSLR").exactly(7).times
-    expect(page.find("table")).to have_content("STRI").exactly(2).times
-    expect(page.find("table")).to have_content("FE").exactly(2).times
```

Comment:
I came across this recently too on my ticket.
My solution was:
```
expect(page).to have_selector("td[text()='ECP']", count: 10)
```
The only advantage to this is that it copes if columns are added/removed from this table, as it doesn't assume a column number.

Reply:
Yeah that's nicer, have changed